src/Evolution/Systems/GeneralizedHarmonic/Equations.cpp

```diff
@@ -8,9 +8,31 @@
 #include "DataStructures/DataVector.hpp"
 #include "DataStructures/Tensor/EagerMath/DotProduct.hpp"
 #include "DataStructures/Tensor/Tensor.hpp"  // IWYU pragma: keep
+#include "Evolution/Systems/GeneralizedHarmonic/Characteristics.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
 #include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
 
 // IWYU pragma: no_forward_declare Tensor
+namespace {
+template <typename FieldTag>
+typename FieldTag::type weight_char_field(
```

Comment:
Should we always use `db::item_type` instead of `::type`? Perhaps @nilsdeppe can you answer this

Reply:
I'll leave this as-is unless there's a consensus that using ::type is problematic

Reply:
`::type` doesn’t work for compute tags. I think for DataBox tags we should always use `db::item_type`, if only to save someone who copies this code some trouble.

Reply:
OK, whenever I refer to tag types, I'll use db::item_type